apps/root/src/pages/dca/create-position/components/landing/chart.tsx

```diff
@@ -0,0 +1,132 @@
+      <path
+        d="M533.779 302.241H532.149L535.813 292.059H537.588L541.252 302.241H539.621L536.743 293.909H536.663L533.779 302.241ZM534.053 298.254H539.343V299.547H534.053V298.254ZM547.44 299.074V294.605H548.932V302.241H547.47V300.919H547.39C547.215 301.326 546.933 301.666 546.545 301.938C546.161 302.206 545.682 302.341 545.108 302.341C544.618 302.341 544.184 302.233 543.806 302.017C543.431 301.799 543.136 301.476 542.921 301.048C542.709 300.62 542.603 300.092 542.603 299.462V294.605H544.089V299.283C544.089 299.803 544.233 300.218 544.522 300.526C544.81 300.834 545.185 300.988 545.645 300.988C545.924 300.988 546.201 300.919 546.476 300.78C546.754 300.64 546.984 300.43 547.167 300.148C547.352 299.866 547.443 299.508 547.44 299.074ZM554.131 305.264C553.524 305.264 553.002 305.184 552.565 305.025C552.13 304.866 551.776 304.656 551.501 304.394C551.226 304.132 551.02 303.845 550.884 303.534L552.162 303.007C552.251 303.153 552.371 303.307 552.52 303.469C552.672 303.635 552.878 303.776 553.136 303.892C553.398 304.008 553.735 304.066 554.146 304.066C554.709 304.066 555.175 303.928 555.543 303.653C555.911 303.381 556.094 302.947 556.094 302.351V300.849H556C555.911 301.012 555.781 301.192 555.612 301.391C555.446 301.59 555.218 301.762 554.926 301.908C554.634 302.054 554.255 302.127 553.788 302.127C553.184 302.127 552.641 301.986 552.157 301.704C551.676 301.419 551.295 301 551.013 300.446C550.735 299.89 550.596 299.205 550.596 298.393C550.596 297.581 550.733 296.885 551.009 296.305C551.287 295.725 551.668 295.281 552.152 294.973C552.636 294.661 553.184 294.505 553.798 294.505C554.272 294.505 554.654 294.585 554.946 294.744C555.238 294.9 555.465 295.082 555.627 295.291C555.793 295.5 555.92 295.684 556.01 295.843H556.119V294.605H557.576V302.41C557.576 303.066 557.424 303.605 557.119 304.026C556.814 304.447 556.401 304.758 555.881 304.961C555.364 305.163 554.78 305.264 554.131 305.264ZM554.116 300.894C554.543 300.894 554.905 300.794 555.2 300.596C555.498 300.393 555.723 300.105 555.876 299.731C556.031 299.353 556.109 298.9 556.109 298.373C556.109 297.86 556.033 297.407 555.881 297.016C555.728 296.625 555.504 296.32 555.21 296.101C554.915 295.879 554.55 295.768 554.116 295.768C553.668 295.768 553.295 295.884 552.997 296.116C552.699 296.345 552.473 296.656 552.321 297.051C552.172 297.445 552.097 297.886 552.097 298.373C552.097 298.874 552.174 299.313 552.326 299.691C552.478 300.069 552.704 300.364 553.002 300.576C553.304 300.788 553.675 300.894 554.116 300.894ZM567.118 292.059V302.241H565.577V293.601H565.517L563.081 295.191V293.72L565.621 292.059H567.118ZM570.215 302.241L574.66 293.451V293.377H569.519V292.059H576.251V293.422L571.821 302.241H570.215Z"
+        fill="#978AA8"
+      />
+      <defs>
+        <linearGradient
+          id="paint0_linear_6629_32270"
+          x1="595.399"
+          y1="88.6885"
+          x2="595.399"
+          y2="265.349"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop stopColor="#791AFF" />
```

Comment:
We can also replace this with violet500, it's the same in both modes, but it doesn't hurt to handle it this way 😄

Reply:
Same as before